docs/source/configuration.rst

```diff
@@ -197,6 +199,23 @@ Options reference
         [chains.foo]
         rpc-url = "http://foo.bar:8545"
 
+   * - ::
+
+        poll-period = TIME
+
+     - Time in seconds which is waited before new events are fetched from the chains after 
+       the last fetch. If a value for a specific chain is provided, it takes precedence for
+       this chain. Default: ``5.0``.
```

Comment:
Nitpick: a bit more specific variant that also highlights this as a global value could look like

```
     - Time in seconds to wait between two consecutive RPC requests for new events.
       The value applies to all chains that don't have the chain-specific poll period defined.
       Default: ``5.0``.
```